examples/cython/.gitignore

```diff
@@ -0,0 +1,6 @@
+build
```

Comment:
How do you feel about moving these to the top-level `.gitignore`?

Reply:
Sure thing; I put in a local ```.gitignore``` since I didn't want to mess with the top-level one. Would it also be useful to put typical virtual environment directories? I waver on that one because even though there are some conventions (e.g., ```venv```), it seems like folks have their own preferences / not totally standardized.

Reply:
I see that the ignores for dynamic libraries are already in the top-level ```.gitignore```.

I think ```build``` is reasonable to go in top-level, but do we want ```*.c```? The Cython convention is to ignore ```*.c``` files in repositories, but include them in distributions so users don't need to install cython to use the package. I put added ```*.c``` for now.